packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Role Assignment
+---
+<ApiLifecycle access="beta" />
+
+# Role Assignment
+Role assignment to principals makes them administrators of your org. Principals can be users or groups of users. When a Role is assigned to a Group, all members of the group automatically have the privileges granted by the role.
+Roles could be one of the [standard roles](#standard-role-assignment), which are provided by default. Alternatively, you can create your own custom roles by choosing from the collection of available [permissions](#permission-types).
+In this page, we discuss the concepts of role assignment through APIs. For more background information see [Custom Admin Roles Help](#https://help.okta.com/en/programs/Content/Topics/betas/closed/custom-admin-role/custom-admin-roles.htm)
+
+## Standard Role Assignment
+The following role types are provided and supported out of the box:
+
+| Role type                               | Label                               | Optional targets                      |
+| :-------------------------------------- | :---------------------------------- | :------------------------------------ |
+| `API_ACCESS_MANAGEMENT_ADMIN`           | API Access Management Administrator |                                       |
+| `APP_ADMIN`                             | Application Administrator           | Apps                                  |
+| `GROUP_MEMBERSHIP_ADMIN`                | Group Membership Administrator      | [Groups](/docs/reference/api/groups/) |
+| `HELP_DESK_ADMIN`                       | Help Desk Administrator             | [Groups](/docs/reference/api/groups/) |
+| `MOBILE_ADMIN`                          | Mobile Administrator                |                                       |
+| `ORG_ADMIN`                             | Organizational Administrator        |                                       |
+| `READ_ONLY_ADMIN`                       | Read-Only Administrator             |                                       |
+| `REPORT_ADMIN`                          | Report Administrator                |                                       |
+| `SUPER_ADMIN`                           | Super Administrator                 |                                       |
+| `USER_ADMIN`                            | Group Administrator                 | [Groups](/docs/reference/api/groups/) |
+
+Standard role assignment is done in two steps:
```

Comment:
Maybe rewrite in active voice, "Perform standard role assignment in these two steps:"